src/content/docs/linter/index.mdx

```diff
@@ -4,180 +4,205 @@ description: How to use the Biome linter.
 
 ```json title="biome.json"
 {
   "linter": {
-    "enabled": true,
     "rules": {
+      "suspicious": {
+        "noDebugger": "off"
+      },
       "style": {
-        "useBlockStatements": "error",
-        "useShorthandArrayType": "error",
-        "noShoutyConstants": "warn"
+        "noShoutyConstants": "warn",
+        "useNamingConvention": "error"
       }
     }
   }
 }
 ```
 
-### Disable a lint rule
+### Rule options
 
-Just add `"off"` as value inside its configuration. For example:
+A few rules have options.
+You can set them by shaping the value of the rule differently.
+
+- `level` will indicate the severity of the diagnostic;
+- `options` will change based on the rule.
 
 ```json title="biome.json"
 {
   "linter": {
-    "enabled": true,
     "rules": {
-      "suspicious": {
-        "noCommentText": "off"
-      },
       "style": {
-        "noUnusedTemplateLiteral": "off"
+        "useNamimgConvention": {
+          "level": "error",
+          "options": {
+            "strictCase": false
+          }
+        }
       }
     }
   }
 }
 ```
 
-### Change the diagnostic severity
 
-Most of Biome's rules will emit an **error**, but you are free to change their severity.
-Just add `"warn"` as value of the rule. Example:
+## Ignoring code
 
-```json title="biome.json"
-{
-  "linter": {
-    "enabled": true,
-    "rules": {
-      "suspicious": {
-        "noCommentText": "warn"
-      }
-    }
-  }
-}
+There are times when a developer wants to ignore a lint rule for a specific line of the code.
+You can achieve this by adding a suppression comment above the line that emits the lint diagnostic.
+
+Suppression comments have the following format:
+
+```js
+// biome-ignore lint: <explanation>
+// biome-ignore lint/suspicious/noDebugger: <explanation>
 ```
 
-This is useful in cases there's being a refactor going on and there's need to make the CI passing.
+Where
 
-## Rule options
+- `biome-ignore` is the start of a suppression comment;
+- `lint` suppresses the linter;
+- `/suspicious/noDebugger`: **optional**, group and name of the rule you want to suppress;
+- `<explanation>` explanation why the rule is disabled
 
-A few rules have options.
-When they do _accept_ some, you can pass them by shaping the value of the rule differently.
+Here's an example:
 
-```json title="biome.json"
+```ts
+// biome-ignore lint: reason
+debugger;
+// biome-ignore lint/suspicious/noDebugger: reason
+debugger;
+```
+
+
+## Ignoring files
+
+Biome doesn't provide ignore comments that ignore an entire file.
+However, you can ignore a file using the Biome configuration file.
+
+You can ignore files for all tools, including the linter,
+using the [`files.ignore` configuration](/guides/how-biome-works#include-and-ignore-explained).
+By default, Biome ignores the [protected files](/guides/how-biome-works#protected-files).
+
+If you want that only the linter to ignore some files, you can use `linter.ignore`:
```

Comment:
A lot of repeated words equals the formatter. I am worried that it could take too much work if a tomorrow will change things, for example, around how we ignore files. 

We should try to keep things in one place, and use links to redirect users.

Reply:
I will try to address the duplication concern in a future PR.